inc/ScaffoldPackageCommand.php

```diff
@@ -110,6 +110,8 @@ public function package( $args, $assoc_args ) {
 		if ( ! Utils\get_flag_value( $assoc_args, 'skip-readme' ) ) {
 			WP_CLI::run_command( array( 'scaffold', 'package-readme', $package_dir ), array( 'force' => $force ) );
 		}
+
+		WP_CLI::run_command( array( 'package', 'install', $package_dir ) );
```

Comment:
I've been playing with this and done some reading, but this doesn't seem to work out. From [the docs](https://wp-cli.org/docs/internal-api/wp-cli-launch-self/#notes):

> Note: While this command does persist a limited set of runtime arguments, it does not persist environment variables. Practically speaking, WP-CLI packages won’t be loaded when using WP_CLI::launch_self() because the launched process doesn’t have access to the current process $HOME.

This was exactly the problem in this case as well (`Package_Command->get_composer_json_path()` couldn't locate the existing packages `composer.json` and tries to create one).